terraform/dashboards/Near/chain-signatures/chain_sig_new.json

```diff
@@ -3450,13 +3351,13 @@
                 "uid": "grafanacloud-prom"
               },
               "editorMode": "code",
-              "expr": "(rate(multichain_num_total_historical_triple_generators_success{environment=~\"$environment\"}[1h]) \n/ rate(multichain_num_total_historical_triple_generators{environment=~\"$environment\"}[1h]))*100",
+              "expr": "(increase(multichain_num_total_historical_triple_generators_success{environment=~\"$environment\"}[1h]) \n/ increase(multichain_num_total_historical_triple_generators{environment=~\"$environment\"}[1h]))*100",
```

Comment:
Seems like this dashboard goes over 100%, I see upwards of 300%

Reply:
![image](https://github.com/user-attachments/assets/77c6265b-154e-4333-8929-05afeb1c30b5)
Last 30 days